docs/releasing.md

```diff
@@ -1,8 +1,36 @@
-## Release guide for maintainers
+### Release guide for maintainers
+
+## Release Policy
+
+### Cadence
+
+We ship twice a month, on the Thursday before the 15th and the Thursday before 1st. These dates are added to the internal Lighthouse calendar.
+
+### Release manager
+
+Rather than a rotation, release manager is appointed. However if the appointed manager is absent, the next engineer in the following list owns it:
+
+    bckenny, paulirish, patrickhulce
+
+Release manager follows the below _Release Process_.
+
+### Release publicity
+
+1. Release mgr copies changelog to a new [Releases](https://github.com/GoogleChrome/lighthouse/releases). Tags and ships it.
+1. Release mgr tells the _LH public_ Hangout chat about the new version.
+1. V writes and publishes the [/updates](https://developers.google.com/web/updates/) blog post
+1. Paul writes the tweet and sends it on [@____lighthouse](https://twitter.com/____lighthouse).
```

Comment:
Question: do we want to wait for the /updates post to be live before Tweeting it out or just stick to the release notes?

Reply:
waiting for /updates seems better. those are more digestible. 
anyone watching the github repo gets an email for a new Release anyway.

Reply:
SGTM. Can we say "writes the tweet with the /updates post"